pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -179,11 +179,15 @@ func (l *listUsersQuery) expand(
 	if enteredCycle(req) {
 		return nil
 	}
-	for _, f := range req.GetUserFilters() {
-		if req.GetObject().GetType() == f.GetType() {
+	for _, userFilter := range req.GetUserFilters() {
```

Comment:
Does it make sense to define some variables above for these intermediate values so we don't have to do `req.Get...` everywhere? For example,

```
objectType := req.GetObject().GetType()
objectID := req.GetObject().GetId()
relation := req.GetRelation()
...
```

Reply:
Yes, makes sense. Competing against another `relation` var lower down so maybe not those literal names.